src/main/resources/com/eprosima/micrortps/idl/templates/CMakeLists.stg

```diff
@@ -24,7 +24,9 @@ endif()
 project(client_examples)
 
 find_package(microcdr REQUIRED)
+$if(examples)$
```

Comment:
the generator must only generate the CMakeList.txt when `-example` option is on. If you want to create extra lines in this file should be using `test` parameter.